indico/core/signals/event/registration.py

```diff
@@ -90,6 +90,14 @@
 ticket for a registrant.
 ''')
 
+is_registration_blocked = _signals.signal('is-registration-blocked', '''
```

Comment:
I don't like the name of it, it sounds more like it's preventing someone from register.
Maybe something like `is_registration_modification_locked` instead?